<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajuda</class>
 <widget class="QDialog" name="Ajuda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pré-Diagnóstico</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Sintomas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>269</width>
      <height>89</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-60</y>
       <width>72</width>
       <height>204</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="SintomaG">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaB">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaF">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaC">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaA">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaH">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaD">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SintomaE">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quais desses sintomas voce possui?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Responda com sim ou não:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="text">
    <string>Verificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>431</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>Genero:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>Idade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>Cidade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <property name="text">
         <string>Bairro:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Masculino</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Feminino</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Outro</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>João Pessoa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>Bayeux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>Cabedelo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>Santa Rita</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="Bairros">
        <property name="font">
         <font>
          <family>MV Boli</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Selecione o seu bairro</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajuda:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Saude Mentall</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Imagens/E:/Felipe/Tec-Pro/Projeto/ProjetoTec/Logo.png</normaloff>:/Imagens/E:/Felipe/Tec-Pro/Projeto/ProjetoTec/Logo.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Covid-19</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Imagens/E:/Felipe/Tec-Pro/Projeto/ProjetoTec/dois.png</normaloff>:/Imagens/E:/Felipe/Tec-Pro/Projeto/ProjetoTec/dois.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="P1">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 1</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="P2">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 2</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="P3">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 3</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="P4">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 4</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="P5">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 5</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="P6">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 6</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="P7">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Questão 7</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="font">
       <font>
        <family>MV Boli</family>
       </font>
      </property>
      <property name="text">
       <string>Nao</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ID_City">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>E:/Felipe/Tec-Pro/Projeto/ProjetoTec/UFPB.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
